perfeed/tools/weekly_summarizer.py

```diff
@@ -53,7 +53,8 @@ async def run(self, users: list[str], repo_name: str, start_of_week: str) -> Non
             self.summarizer.run(repo_name, pr_number) for pr_number in pr_numbers
         ]
 
-        summaries = await asyncio.gather(*summary_futures, return_exceptions=True)
+        resolved_summaries = await asyncio.gather(*summary_futures, return_exceptions=True)
+        summaries = [resolved_summary[0] for resolved_summary in resolved_summaries if not isinstance(resolved_summary, BaseException)]
```

Comment:
a heads up that i will need metadata for weekly as well. will make change on my end

Reply:
OK